src/js/app.js

```diff
@@ -3,9 +3,9 @@
  *
  * This is where you write your app.
  */
-
-var UI = require('ui');
-var Vector2 = require('vector2');
+require('./main');
+var UI = require('./ui');
+var Vector2 = require('./lib/vector2');
```

Comment:
Is there any way to version Pebble.js so only new projects will use the SDK JS processing? 

Barring that, I'd be inclined to hold off on merging this into master, in favor of getting Pebble.js working as a Pebble Package (https://github.com/cjwilliams/pebblejs/compare/task/convert-to-modern...cjwilliams:feature/pebble-package, which is having issues with the PebbleJS icon), and releasing the package on npm with a major version bump (deprecating the non-package Pebble.js)